modules/tinymce/src/plugins/accordion/main/ts/core/readonly/attribute/AttributeUtils.ts

```diff
@@ -0,0 +1,33 @@
+import { accordionTemporaryOpenAttribute } from '../../Identifiers';
+
+export interface AttributeOperationAdapter<T> {
```

Comment:
I see that you try to use the same code but for the ast and the regular dom. But I wonder if this abstraction is the same size as just duplicating the code for toggling those attributes?
Does this add extra complexity with little gain? If this module was heavy then I would get that it would give us more but right now it's just a couple of lines so to me it mostly adds complexity.
Or do you think this part of the code will grow so that it would make sense to have an abstraction for the two tree models?